api/app/src/external/commonwell/document/document-query-sandbox.ts

```diff
@@ -116,3 +132,63 @@ export async function sandboxGetDocRefsAndUpsert({
 
   return result;
 }
+
+function addSandboxFields(docRef: DocumentReference): DocumentReference {
+  if (docRef.content) {
+    const fileExt = docRef.content[0]?.attachment?.contentType?.split("/")[1]?.trim();
+    const randomIndex = Math.floor(Math.random() * randomDates.length);
+    // Add a random date to the sandbox document
+    if (docRef.content[0]?.attachment) {
+      docRef.content[0].attachment.creation = randomDates[randomIndex];
+    }
+
+    if (randomIndex < 3) {
+      docRef.status = "current";
+    } else {
+      docRef.status = "superseded";
+    }
+
+    if (fileExt === "xml") {
+      docRef.description = "C-CDA R2.1 Patient Record";
+      docRef.type = {
+        coding: [
+          {
+            system: "http://loinc.org/",
+            code: "34133-9",
+            display: "Summarization of episode note",
+          },
+        ],
+      };
+    } else if (fileExt === "pdf") {
+      (docRef.description = "Physical Examination"),
+        (docRef.type = {
+          coding: [
+            {
+              display: "Encounter",
+            },
+          ],
+        });
+    } else if (fileExt === "tif") {
```

Comment:
this should be `else if (fileExt === "tif" || fileExt === "tiff") {`

https://en.wikipedia.org/wiki/TIFF